runtime/browser/xwalk_render_message_filter.h

```diff
@@ -17,12 +17,17 @@ class XWalkRenderMessageFilter : public content::BrowserMessageFilter {
   explicit XWalkRenderMessageFilter(int process_id);
 #endif
   bool OnMessageReceived(const IPC::Message& message) override;
-
- private:
-  void OnOpenLinkExternal(const GURL& url);
 #if defined(OS_ANDROID)
   void OnSubFrameCreated(int parent_render_frame_id, int child_render_frame_id);
+  void OnShouldOverrideUrlLoading(int routing_id,
```

Comment:
it would be nice to keep these two OnXXX as private as they are both your private implementation.